tools/recipes/pahole.recipe

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+download() {
+    git clone -b 'v1.20' --single-branch --depth 1 git://git.kernel.org/pub/scm/devel/pahole/pahole.git pahole_src
+    echo "Pahole download completed"
```

Comment:
There is no need for extra echo, "lisa-build-asset" already logs when it runs a given action.

Reply:
Resolved in new version